tests/Transports.AspNetCore.Tests/Transports.AspNetCore.Tests.csproj

```diff
@@ -1,19 +1,49 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <Import Project="../../Tests.props" />
 
-  <PropertyGroup>
-    <TargetFrameworks>net6;net5;netcoreapp3.1</TargetFrameworks>
+  <PropertyGroup Condition="'$([System.Runtime.InteropServices.RuntimeInformation]::IsOSPlatform($([System.Runtime.InteropServices.OSPlatform]::Windows)))' != 'true'">
```

Comment:
Let's solve this task by [Microsoft.NETFramework.ReferenceAssemblies](https://www.nuget.org/packages/Microsoft.NETFramework.ReferenceAssemblies.net461/) 

https://andrewlock.net/using-reference-assemblies-to-build-net-framework-libararies-on-linux-without-mono/

Reply:
Sounds good; I'll add an issue so I can investigate and test

Reply:
#845